skycoin-api/Makefile

```diff
@@ -80,11 +81,18 @@ libskycoin-crypto.so: skycoin_crypto.o $(OBJS)
 libskycoin-crypto.a: skycoin_crypto.o $(OBJS)
 	$(AR) rcs libskycoin-crypto.a skycoin_crypto.o $(OBJS)
 
+libskycoin-wrapper.o: libskycoin-wrapper.c
```

Comment:
Please explain (briefly) in README.md how running `libskycoin` test suite works and how this is used for running compatibility test suite.